angrmanagement/ui/workspace.py

```diff
@@ -25,14 +28,20 @@ def __init__(self, main_window, instance):
         self.views_by_category = defaultdict(list)
         self.views = [ ]
         self.dockable_views = [ ]
+        self.dockable_views2 = [ ]
         self.view_to_dockable = { }
+        self.last_unsplit_view = None
+        self.split_tab_id = None
+        self.is_split = 0
+        self.splitter_state = QSettings()
+        self.splitter_state.setValue("splitterSizes", self._main_window.central_widget_main.saveState())
 
         #
         # Initialize font configurations
         #
         Conf.init_font_config()
 
-        default_tabs = [
+        self.default_tabs = [
```

Comment:
What's the rationale behind this change?

Reply:
I forgot to add that most (if not all) default tabs can be closed by user as needed. So we should use the new view manager to get a reference to the currently open view.

Reply:
I added self so it can be accessed by other methods

Reply:
Gotcha. I missed that. Thanks.